src/man/sss_override.8.xml

```diff
@@ -44,6 +44,12 @@
             <emphasis>sss_override</emphasis> prints message when a restart is
             required.
         </para>
+        <para>
+            <emphasis>NOTE:</emphasis> The options provided in this man page
+            only work with <quote>ldap</quote> and <quote>AD</quote> <quote>
+            id_provider</quote>.  Additionally IPA overrides can be managed
+            centrally on the IPA server.
```

Comment:
Hi,

thanks for the update. I think the word `Additionally` can be removed from the last sentence. @justin-stephenson or @sidecontrol can one of you check this from a native speaker point of view?

bye,
Sumit

Reply:
Agreed with removing 'Additionally', otherwise LGTM